tutorials/getstarted.jl

```diff
@@ -0,0 +1,1051 @@
+### A Pluto.jl notebook ###
+# v0.19.11
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ c96935ca-6bda-466d-ad29-b40c19f55392
+using Manifolds
+
+# ╔═╡ 41cbc7c8-3a39-11ed-292e-0bb253a3b2f3
+md"""
+# 🚀 Get Started with `Manifolds.jl`
+
+This is a short overview of [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/).
+
+This tutorial is rendered from a pluto notebook, so you can also open the file [tutorials/getstarted.jl](https://github.com/JuliaManifolds/Manifolds.jl/tree/master/tutorials/getstarted.jl) in Pluto and work on this tutorial interactively.
+
+As usual, if you want to install the package, just type
+
+```
+] add Manifolds
+```
+
+in Julia REPL or use
+
+```
+using Pkg; Pkg.add("Manifolds");
+```
+
+before the first use. Then load the package with
+"""
+
+# ╔═╡ 9d16efde-bd95-46d9-a659-5420fe860699
+md"""
+Since the package heavily depends on [`ManifoldsBase.jl`](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/) we will sometimes also link to the interface definition of functions in the interface and mark this with 🔗. When referring to [Wikipedia](https://en.wikipedia.org/), the link is marked with 📖.
+"""
+
+# ╔═╡ b34d2b6c-907e-45b3-9b62-445666413b26
+md"""
+## Contents
+* [Using the library of manifolds](#Using-the-Library-of-Manifolds)
+* [implementing generic functions](#Implementing-generic-Functions)
+* Allocating and in-place computations
+* decorating a manifold
```

Comment:
I'd suggest adding a short section about differences between chart-based and intrinsic approaches to computations on a manifold. Maybe even one or two paragraphs that would explain that while the standard textbook approach is to have an atlas and do everything in a chart, the package focuses on intrinsic computations based in an embedding. I remember some people reporting being confused by that.

After #533 is merged we can turn that torus example into a section about working in charts.

Reply:
Uff, until now I would have no clue how to explain that. Even when I started with the second section it not only got far too long (I think), it also is quite technical (and that part is easier than charts I think).

Reply:
OK, I will suggest a text later. I think it's fine that not everyone interested in Manifolds.jl would understand such explanation because it may be important for people familiar with charts.

Reply:
yes absolutely. I came from a differential geometry background and after implementing a couple toy packages that were charts based and I remember it took me a while to realize that Manifolds.jl was working with embedded manifolds. I think a note would clear that up for people interested in it, and it's okay if for some (though I suspect not many) is a bit obscure

Reply:
I just finished a decorating section and would like to do the charts with the example from Mateusz – I can add a link to that here. I think a “Further Reading” section would be a good final one here.

Reply:
OK, do you think it would make sense to rewrite my example as a docs page or a Pluto notebook?

Reply:
I will rewrite yours as a Pluto notebook, sure :)

Reply:
Cool, thanks :slightly_smiling_face: